Makefile

```diff
@@ -1,49 +1,22 @@
-# OASIS_START
-# DO NOT EDIT (digest: a3c674b4239234cbbe53afe090018954)
-
-SETUP = ocaml setup.ml
-
-build: setup.data
-	$(SETUP) -build $(BUILDFLAGS)
-
-doc: setup.data build
-	$(SETUP) -doc $(DOCFLAGS)
-
-test: setup.data build
-	$(SETUP) -test $(TESTFLAGS)
+VERSION=$(shell dune exec src/version.exe)
+NAME=murmur3-$(VERSION)
```

Comment:
I changed the name from `ocaml-murmur3` to `murmur3` - is that alright?

Reply:
I decided to change it back to ocaml-murmur3.

Reply:
better not change to not cause breakage, yes

Reply:
Yep, already did (latest version should be correct)